tests/ReflectorTest.php

```
@@ -0,0 +1,427 @@
+        $this->expectExceptionMessage(
+            Message::REFLECTOR_PROPERTY_NOT_FOUND->getMessage('missing', ReflectionFixture::class),
+        );
+
+        Reflector::property(ReflectionFixture::class, 'missing');
+    }
+
+    public function testPropertyTypeNamesReturnsEmptyArrayForUntypedProperty(): void
+    {
+        self::assertSame(
+            [],
+            Reflector::propertyTypeNames(ReflectionFixture::class, 'untyped'),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PHPForge\Helper\Reflector' in method 'testPropertyTypeNamesReturnsEmptyArrayForUntypedProperty'. (undefined)

(StaticAccess)